### ISSUE TITLE
performance measurement

### ISSUE THREAD

**Issue by onlinebizsoft** [yireo/Yireo_DisableLog2#12]
Is this extension actually improving any performance? I understand it only remove event log for customer login and customer logout which doesn't happen often even for a busy store?

**Comment by jissereitsma** [comment]
Yes, it only removes logging to the `customer_log` table. In the past, there used to be many more logging features (the `report_` tables). But as of yet, it is less of a burden. If those logs are costing you performance, is depending on your shop.
